go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/asaskevich/govalidator v0.0.0-20200428143746-21a406dcc535 // indirect
 	github.com/go-openapi/analysis v0.19.10
-	github.com/go-openapi/errors v0.19.6
+	github.com/go-openapi/errors v0.19.7-0.20200915043824-00a6ebeed93c
```

Comment:
We need to properly update version. Once errors pkg tag is created.

Reply:
it's tagged you can get it with ```go get github.com/go-openapi/errors@v0.19.7```

Reply:
Updated. Thanks.